test/tests/meshgenerators/flip_sideset_generator/tests

```diff
@@ -0,0 +1,44 @@
+[Tests]
+    [flux_2D]
+      type = 'CSVDiff'
+      input = 'flux_flip_2D.i'
+      csvdiff = 'flux_flip_2D_out.csv'
+      recover = false
+      issues = '#25528'
+      design = 'FlipSidesetGenerator.md'
```

Comment:
Take these out of each test and put them under the [Tests] top level, i.e.,

```
[Tests]
  issues = '#25528'
  design = 'FlipSidesetGenerator.md'
  ...
```

as they are relevant to all tests